### PR TITLE
Update video on home page

### PR DESCRIPTION
**Pull request by travisbeckham**

### CODE REVIEW

File: linkerd.io/content/_index.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ params:
         url: https://buoyant.io/case-studies/schenker/
   about:
     title: What is Linkerd?
```
Comment (kflynn) [inline linkerd.io/content/_index.md:111]:
```suggestion
    title: Linkerd: The Lightest, Fastest, Most Secure Service Mesh on the Planet
```

???

**Review by kflynn — CHANGES_REQUESTED** [review]
@CathPag, think we should change the title too?

File: linkerd.io/content/_index.md (hunk 1 above)
Comment (CathPag) [reply linkerd.io/content/_index.md:111]: Sure

**Review by kflynn — APPROVED** [review]
Hit it! 🙂

**Pull request merged**